### DISCUSSION TITLE
quarto use cross-reference for reactable

### DISCUSSION THREAD

**Discussion by Liripo** [quarto-dev/quarto-cli#782] [Q&A] [answered]
Is there a similar way for cross-referencing of htmlwidgets like reactable？[https://github.com/glin/reactable/issues/15](https://github.com/glin/reactable/issues/15)

**Comment by jjallaire** [comment, marked as answer]
Currently since the HTML output by reactable isn't actually a table, we can only cross-reference it as a "figure":

````
```{r}
#| label: fig-table
#| fig-cap: "My Table"

library(reactable)
reactable(iris)
```
````

**Reply by Liripo** [reply]
Thanks for your answer. if it would be nice to have a cross-reference to the htmlwidgets.

**Reply by jjallaire** [reply]
Okay, we'll try to make this work sometime soon.

**Reply by canholyavkin** [reply]
Hello @jjallaire. Is there any update on cross-referencing on reactable objects?

**Comment by cderv** [comment]
Related issues: https://github.com/quarto-dev/quarto-cli/issues/1084
